src/main/java/io/fabric8/elasticsearch/plugin/ConfigurationSettings.java

```diff
@@ -66,6 +66,9 @@
 	 */
 	/** OpenShift settings here **/
 	static final String OPENSHIFT_ES_ACL_DELAY_IN_MILLIS = "io.fabric8.elasticsearch.acl.sync_delay_millis";
+	static final String OPENSHIFT_ES_KIBANA_SEED_MAPPINGS_APP = "io.fabric8.elasticsearch.kibana.mapping.app";
+	static final String OPENSHIFT_ES_KIBANA_SEED_MAPPINGS_OPERATIONS = "io.fabric8.elasticsearch.kibana.mapping.ops";
+	static final String OPENSHIFT_ES_KIBANA_SEED_MAPPINGS_EMPTY = "io.fabric8.elasticsearch.kibana.mapping.empty";
```

Comment:
So how will a user configure ES to use a different file?  Something like this in `elasticsearch.yml`?
```
io.fabric8.elasticsearch.kibana.mapping.app: /full/path/to/my/app.json
```
?